src/numa.cpp

```diff
@@ -34,7 +38,11 @@ numa_tools::size_type numa_tools::page_size_ = sysconf(_SC_PAGESIZE);
 
 namespace
 {
+#ifdef HWMALLOC_NUMA_AVAILABLE
```

Comment:
I would prefer a separate implementation of the stub instead of interspersing the code with ifdefs. The advantage would be
- better readability
- get rid of `(void)` casts (which you do to suppress compiler warnings I guess?
- no need for assert (same reason)

Reply:
I agree, I'll do that.